index.js

```diff
@@ -1,17 +1,65 @@
 class SortedList {
-  constructor() {}
+   
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items = this.items.sort( (a,b) => {
+      return a-b;
+    });
+    this.length++;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < 0 || pos > this.items.length) {
+      throw new Error ('OutOfBounds');
+    } else {
+      return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length === 0) {
+      throw new Error('EmtySortedList');
+    } else {
+      return Math.max(...this.items);
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.items.length === 0) {
+      throw new Error('EmtySortedList');
+    } else {
+      return Math.min(...this.items);
+    }
+  }
 
-  sum() {}
+  sum() {
+    let sum = 0;
+    if (this.items.length === 0) {
+      return sum;
+    } else {
+      sum = this.items.reduce((acc, elem) => {
+        return acc += elem;
+      }, 0);
+      return sum;
+    }
+  }
 
-  avg() {}
+  avg() {
+    let sum = 0;
+    if (this.items.length === 0) {
+      throw new Error('EmtySortedList');
+    } else {
+      sum = this.items.reduce((acc, elem) => {
```

Comment:
Once you already have the sum() function above you could just used it and divide it by the length.